packages/web-react/src/components/Footer/Footer.tsx

```diff
@@ -2,16 +2,25 @@
 
 import classNames from 'classnames';
 import React from 'react';
+import { BackgroundColors } from '../../constants';
 import { useStyleProps } from '../../hooks';
 import { SpiritFooterProps } from '../../types';
+import { useFooterStyleProps } from './useFooterStyleProps';
 
-const Footer = (props: SpiritFooterProps): JSX.Element => {
-  const { children, ...restProps } = props;
+const defaultStyleProps: Partial<SpiritFooterProps> = {
+  backgroundColor: BackgroundColors.SECONDARY,
+  paddingBottom: 'space-1200',
```

Comment:
**suggestion (if-minor):** Can we also use a constant instead of the hardcoded value, please?